keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,418 @@
+
+#### Basic Networking
+
+- Ability to mount host volumes
+- Ability to access linux container IPs *by service IP* from windows containers
+- Ability to access windows container IPs *by service IP* from linux containers
+- Ability to access linux container IPs *by NodePort IP* from windows containers
+- Ability to access windows container IPs *by NodePort IP* from linux containers
+- Ability to access linux container IPs *by pod IP* from windows containers
+- Ability to access windows container IPs *by pod IP* from linux containers
+- Ability to mount a hostPath storage volume from a Kubelet into a pod and write to it. 
+- Ability to mount a hostPath storage volume from a Kubelet into a pod and read from it. 
+- Ability to schedule multiple containers, with distinct IP addresses, on the same node.
+- Ability to reboot all of the Windows nodes in a cluster while running windows pods, in such a way that recovery of these deleted pods occurs in the scope of a few minutes.
+- Ability to delete and recreate services in such a way that load balancing rules for pods are recovered by whatever chosen service proxy is being utilized.
+- Ability to delete and recreate pods for StatefulSets which preserve their ability to serve as routed endpoints for services, while also having unchanging IP addresses.
+- Ability to access internal pods by internal Kubernetes DNS service endpoints.
+- Ability to access external services by Kubernetes DNS services endpoints (for non-airgapped clusters).
+- Ability to access internal pods through custom DNS records, injected by the Kubernetes pod Specification.
+- Ability to route services from pods from the EndpointSlice API.
+
+#### Basic Service accounts and local storage
+
+- Ability to rotate service accounts for running pods, with the APIServer access capacity of the pod remaining unchanged over the long term.
+- Ability to read and write shared files on a single Kubernetes node between three running Windows containers, simultaneously.
+
+#### Basic Scheduling
+
+- Ability to schedule pods with CPU and Memory limits demonstrably honored over time.
+- Ability to demonstrate the pods requesting more CPU and Memory then available are left in the Pending state over time.
+
+#### Basic performance
+
+- Ability to reschedule the a deployment Pod, 50 times in a row, with continuous deletions in the background.
+- Ability to route traffic to 10 pods behind a common ClusterIP service endpoint.
+- Ability to route traffic to 10 pods behind a common NodePort service endpoint.
+
+We do not include GMSA support in the basic definition of WindowsConformance, because it is known to require other cluster components outside of Kubernete's Control.
```

Comment:
```suggestion
We do not include GMSA support in the basic definition of Windows Conformance because it is known to require other cluster components outside of Kubernetes's Control.
```